qa/acceptance-tests/src/test/java/io/camunda/it/auth/MappingRuleAuthorizationIT.java

```diff
@@ -38,7 +39,7 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.condition.DisabledIfSystemProperty;
 
-@MultiDbTest
+@MultiDbTest(DatabaseType.OS)
```

Comment:
❓ I think we weren't supposed to check those in?

Reply:
You didn't even give me like 5 seconds to fix it, come on :(